utils/manifest-utils/src/keystore/x509_keystore.rs

```diff
@@ -63,6 +63,7 @@ impl X509CertData {
 pub(super) struct AddX509Response {
     pub loaded: Vec<X509>,
     pub skipped: Vec<X509>,
+    pub leaf_cert_ids: Vec<String>,
```

Comment:
Why do we have multiple leafs?
From my understanding when I add whole chain to keystore- there is only one leaf for it

Reply:
Discussed on discord. The method can be used to add multiple chains, so it may return multiple leaf cert ids.